payments/mobile-pay/other-features.md

```diff
@@ -16,7 +16,819 @@ sidebar:
+| └➔&nbsp;`acquirerTransactionTime` | `string` | The ISO-8601 date and time of the acquirer transaction.                                      |
+| └➔&nbsp;`authenticationStatus`    | `string` | `Y`, `A`, `U` or `N`. Indicates the status of the authentication.                            |
+| └➔&nbsp;`transaction`             | `object` | The object representation of the generic transaction resource.                               |
+
+### Create authorization transaction
+
+The authorization transaction is initiated by redirecting the end-user/consumer
+to the hyperlink returned in the `redirect-authorization` request.
+
+## Captures
+
+The `captures` resource lists the capture transactions (one or more) on a
+specific payment.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/mobilepay/payments/{{ page.paymentId }}/captures HTTP/1.1
+Host: api.payex.com
+Authorization: Bearer <MerchantToken>
+Content-Type: application/json
+```
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/mobilepay/payments/{{ page.paymentId }}",
+    "captures": {
+        "id": "/psp/mobilepay/payments/{{ page.paymentId }}/captures",
+        "captureList": [{
+            "id": "/psp/mobilepay/payments/{{ page.paymentId }}/captures/{{ page.transactionId }}",
+            "transaction": {
+                "id": "/psp/mobilepay/payments/{{ page.paymentId }}/transactions/{{ page.transactionId }}",
+                "created": "2016-09-14T01:01:01.01Z",
+                "updated": "2016-09-14T01:01:01.03Z",
+                "type": "Capture",
+                "state": "Completed",
+                "number": 1234567890,
+                "amount": 1000,
+                "vatAmount": 250,
+                "description": "Test transaction",
+                "payeeReference": "AH123456",
+                "failedReason": "",
+                "isOperational": false,
+                "operations": []
+            }
+        }]
+    }
+}
+```
+
+{:.table .table-striped}
+| Property                | Type     | Required                                                                     | Description |
+| :---------------------- | :------- | :--------------------------------------------------------------------------- | :---------- |
+| `payment`               | `string` | The relative URI of the payment this list of capture transactions belong to. |
+| `captures`              | `object` | The current `captures` resource.                                             |
+| └➔&nbsp;`id`            | `string` | The relative URI of the current `captures` resource.                         |
+| └➔&nbsp;`captureList`   | `array`  | The array of capture transaction objects.                                    |
+| └➔&nbsp;`captureList[]` | `object` | The capture transaction object described in the `capture` resource below.    |
```

Comment:
We should describe the `transaction` object here as well, perhaps with an include?

Reply:
Using includes and such in tables results in a new table where the code is written.